Update Filter_Types.md

LRahlff opened the pull request:
update to no_input_handling branch in fish

LRahlff committed:
Update Filter_Types.md

Doralitze commented:
This isn't going to work as the scheduling algorithm does not support left empty inputs. Otherwise we would need an algorithm with higher time complexity which we cannot afford. The solution must be implemented with the project editor where we have a set of all connections and therefor an O(1) check if a connection is actually empty.

Edit: Never mind, having this in the Docs is useful for implementing default values in the Editor.

LRahlff committed:
Update Filter_Types.md

changed default value for trigonometrics, so with the time input it is not that fast, now 3.6 seconds for each period

Doralitze reviewed (approved):
LGTM
